app/Bundle.php

```diff
@@ -179,33 +178,63 @@ public function syncVouchers(array $voucherCodes)
     }
 
     /**
-     * Return the Centre it was allocated to
+     * Get the Registration this bundle is for
      *
      * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
      */
-    public function centre()
+    public function registration()
+    {
+        return $this->belongsTo(Registration::class);
+    }
+
+    /**
+     * The vouchers in this Bundle
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function vouchers()
+    {
+        return $this->hasMany(Voucher::class);
+    }
+    /**
+     * Return the Carer it was disbursed to
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
+     */
+    public function collectingCarer()
+    {
+        return $this->belongsTo(Carer::class);
+    }
+
+    /**
+     * Return the Centre it was disbursed to
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
+     */
+    public function disbursingCentre()
     {
         return $this->belongsTo(Centre::class);
     }
 
     /**
-     * Get the Registration this bundle is for
+     * Return the Carer it was disbursed to
```

Comment:
Carer doesn't match CentreUser I think should be 'Centre User who disbersed the bundle'?